Cargo.toml

```diff
@@ -29,16 +29,13 @@ amethyst_input = { path = "amethyst_input", version = "0.1" }
 cgmath = "0.14"
 cpal = "0.4"
 crossbeam = "0.3"
-dds-rs = "0.4"
 derivative = "1.0"
 fnv = "1.0"
 futures = "0.1.14"
 gfx = "0.16"
 num_cpus = "1.2"
 genmesh = "0.4"
 imagefmt = "4.0"
-serde = "1"
-serde_derive = "1"
```

Comment:
Pretty sure this breaks things. This crate uses serde to create ron files.

Reply:
... Apparently Travis thinks it's Ok?  I guess I need to look into this further but how can you implement Serialize without serde?

Reply:
Oh woops my bad, I understand now.